### PR TITLE
Reduce nestiness of `PreCloudInitIso` method

### PR DESCRIPTION
**Pull request by VirrageS**
**What this PR does / why we need it**: Reduces complexity/nestiness of `PreCloudInitIso` method by reverting `if` conditions.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*: None

**Special notes for your reviewer**: There are no logic changes - this is strictly refactoring MR.

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by VirrageS** [comment]
@rthallisey could you take a look?

**Review by rthallisey — COMMENTED** [review]
One question.  Rest looks ok to me

File: pkg/hooks/manager.go [hunk 1]
```diff
@@ -258,75 +258,82 @@ func (m *hookManager) onDefineDomainCallback(callback *callBackClient, domainSpe
 }
 
 func (m *hookManager) PreCloudInitIso(vmi *v1.VirtualMachineInstance, cloudInitData *cloudinit.CloudInitData) (*cloudinit.CloudInitData, error) {
-	if callbacks, found := m.CallbacksPerHookPoint[hooksInfo.PreCloudInitIsoHookPointName]; found {
-		for _, callback := range callbacks {
```
[the remark this answers is not in the document]
Comment (VirrageS) [reply pkg/hooks/manager.go:262]: Nope, at L325 there is `return resultData, nil` which returns on the first callback, so even if it is possible to define multiple callbacks, only one is getting executed. And in my rewrite, I also follow this convection - I don't want to introduce any user-facing changes.

**Comment by VirrageS** [comment]
@xpivarc could you take a look to approve?

**Pull request closed without merging**